src/ConnectedVMware/test/Az.ConnectedVMware-TestResults.xml

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<test-results xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="nunit_schema_2.5.xsd" name="Pester" total="7" errors="0" failures="7" not-run="0" inconclusive="0" ignored="14" skipped="0" invalid="0" date="2023-10-01" time="09:30:16">
+  <environment cwd="C:\workspace\new-azure-powershell\src\ConnectedVMware" clr-version="Unknown" user="sanmishra" nunit-version="2.5.8.0" os-version="10.0.25951" machine-name="SANMISHRA-D2" user-domain="FAREAST" platform="Microsoft Windows 11 Enterprise Insider Preview|C:\WINDOWS|\Device\Harddisk1\Partition4" />
+  <culture-info current-culture="en-US" current-uiculture="en-US" />
+  <test-suite type="TestFixture" name="Pester" executed="True" result="Failure" success="False" time="3.3181" asserts="0" description="Pester">
+    <results>
+      <test-suite type="TestFixture" name="C:\workspace\new-azure-powershell\src\ConnectedVMware\test\AzConnectedVMwareCluster.Tests.ps1" executed="True" result="Failure" success="False" time="0.8865" asserts="0" description="C:\workspace\new-azure-powershell\src\ConnectedVMware\test\AzConnectedVMwareCluster.Tests.ps1">
```

Comment:
> result="Failure" success="False" [](http://example.com/codeflow?start=153&length=33)

Don't you need to fix these failed tests?

Reply:
fixed.